astropy/io/ascii/core.py

```diff
@@ -162,10 +162,18 @@ class BaseInputter(object):
     * List of strings
     """
     def get_lines(self, table):
-        """Get the lines from the ``table`` input.
+        """
+        Get the lines from the ``table`` input.
+
+        Parameters
+        ----------
+        table : `~astropy.table.Table`
```

Comment:
This accepts a wide variety of inputs, but *not* an astropy Table.  From http://astropy.readthedocs.org/en/latest/io/ascii/read.html#parameters-for-read, it's a little complicated, which is why the original is so vague:
```
There are four ways to specify the table to be read:

- Name of a file (string)
- Single string containing all table lines separated by newlines
- File-like object with a callable read() method
- List of strings where each list element is a table line

The first two options are distinguished by the presence of a newline in the string. This assumes that valid file names will not normally contain a newline.
```